GNUmakefile

```diff
@@ -14,6 +14,9 @@ test: fmtcheck
 testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 180m
 
+debugacc: fmtcheck
+	TF_ACC=1 dlv test $(TEST) --headless --listen=:2345 --api-version=2 -- -test.v $(TESTARGS)
```

Comment:
👍  this is a helpful one to expose; we should probably document how to use it at some point